deploy/infra/create_cpu_batch_account_pool.bicep

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Is this tested end to end (bicep deployment + pipeline run) ?

In the end-to-end testing, we should test the option b for loading the docker image (image loaded to the ACR deployed as part of this deployment) and without providing register credentials.

Reply:
This is tested for docker image on acr.